### PR TITLE
Implement clean code in Program.cs

### PR DESCRIPTION
**Pull request by TC-ibautista**

### CODE REVIEW

File: csharp/Clean Code/ToDo/Program.cs [hunk 1]
```diff
@@ -5,33 +5,48 @@ namespace ToDo
 {
     internal class Program
     {
-        public static List<string> TL { get; set; }
+        public static List<string> TaskList { get; set; } = new List<string>();
```
Comment (TC-lSanchez) [inline csharp/Clean Code/ToDo/Program.cs:9]: Good!

File: csharp/Clean Code/ToDo/Program.cs [hunk 2]
```diff
@@ -5,33 +5,48 @@ namespace ToDo
...
 
         static void Main(string[] args)
         {
-            TL = new List<string>();
-            int variable = 0;
+            const int addTask = 1;
+            const int removeTask = 2; 
+            const int consultTasks = 3; 
+            const int exit = 4;
+            int menuOption = 0;
```
Comment (TC-oRovira) [inline csharp/Clean Code/ToDo/Program.cs]: When using constants, you wanna follow naming convention for constants in c# which would be PascalCase, additionally, constants can be placed in a different class file containing only constants for easier reference.

However, for you concrete use case I would recommend an enum which is easier to read and maintain for menu options like this:
```suggestion
private enum MenuOption
{
    AddTask = 1,
    RemoveTask = 2,
    ConsultTasks = 3,
    Exit = 4
}
```

File: csharp/Clean Code/ToDo/Program.cs [hunk 3]
```diff
@@ -5,33 +5,48 @@ namespace ToDo
...
+                        Console.WriteLine("See you!");
+                        break;
+
+                    default:
+                        Console.WriteLine("Option not allowed, try again");
+                        break;
                 }
-            } while (variable != 4);
+            } while (menuOption != exit);
         }
```
Comment (TC-oRovira) [inline csharp/Clean Code/ToDo/Program.cs]: To go with it, I'd also add something like this:
```C#
    public static void Main(string[] args)
        {
            MenuOption menuOption;
            do
            {
                menuOption = ShowMainMenu();
                HandleMenuOption(menuOption);
            } while (menuOption != MenuOption.Exit);
        }
```

File: csharp/Clean Code/ToDo/Program.cs [hunk 4]
```diff
@@ -41,70 +56,93 @@ public static int ShowMainMenu()
             Console.WriteLine("3. Pending tasks");
             Console.WriteLine("4. Exit");
 
-            // Read line
-            string line = Console.ReadLine();
-            return Convert.ToInt32(line);
+            string optionSelected = Console.ReadLine();
+            return Convert.ToInt32(optionSelected);
         }
```
Comment (TC-oRovira) [inline csharp/Clean Code/ToDo/Program.cs:60]: And this:

```C#
    private static MenuOption ShowMainMenu()
        {
            Console.WriteLine("1. Add Task");
            Console.WriteLine("2. Remove Task");
            Console.WriteLine("3. Consult Tasks");
            Console.WriteLine("4. Exit");
            Console.Write("Select an option: ");

            if (Enum.TryParse(Console.ReadLine(), out MenuOption menuOption) && Enum.IsDefined(typeof(MenuOption), menuOption))
            {
                return menuOption;
            }

            Console.WriteLine("Invalid option, please try again.");
            return ShowMainMenu();
        }
```

File: csharp/Clean Code/ToDo/Program.cs [hunk 5]
```diff
@@ -5,33 +5,48 @@ namespace ToDo
...
+                        Console.WriteLine("Option not allowed, try again");
+                        break;
                 }
-            } while (variable != 4);
+            } while (menuOption != exit);
         }
+        
         /// <summary>
-        /// Show the main menu 
+        /// Shows the tasks options to redirect to the selected menu
```
Comment (TC-oRovira) [inline csharp/Clean Code/ToDo/Program.cs:45]: Regarding comments, I think the general rule is to keep method and variable names meaningful and descriptive and avoid doing code that needs to be explained in a comment, so for the most part of your code you wanna avoid commenting things and instead focus on making it very simple (in all possible ways). For the times where code is not simple/clear enough, is a great idea to add comments to help the reader

**Review by TC-oRovira — COMMENTED** [review]
Let me know what do you think about this suggestions 👍  and keep up the good work

**Pull request closed without merging**